src/components/PopupText/PopupText.tsx

```diff
@@ -5,22 +5,61 @@ import {
   CALENDLY_STYLESHEET_SOURCE,
 } from "../../constants";
 
+
 export interface Props {
   url: string;
   text: string;
+  prefill?: Prefill;
+}
+
+type Optional<T extends object> = {
```

Comment:
Can you move the `Optional` and `Prefill`types to `calendly.tsx` since these will be shared by all 3 components?